EventsExpress/ClientApp/src/components/login/login.js

```diff
@@ -7,18 +7,19 @@ import Modalwind2 from '../recoverPassword/modalwind2';
 import GoogleLogin from '../../containers/GoogleLogin';
 import LoginFacebook from '../../containers/FacebookLogin';
 import TwitterLogin from '../../containers/TwitterLogin';
+import ErrorMessages from '../shared/errorMessage';
 
 const { validate, renderTextField, asyncValidate } = Module;
 
 class Login extends Component {
   openModal = () => (<Modalwind2 />)
```

Comment:
This method is not used anywhere? Probably remove it?

Reply:
I guess it is for "Forgot password" flow